mesa/__init__.py

```diff
@@ -9,8 +9,12 @@
 from mesa.model import Model
 from mesa.agent import Agent
 
+import mesa.time as time
+import mesa.space as space
+import mesa.visualization_flat as visualization
+from mesa.datacollection import DataCollector
```

Comment:
This is the line in this PR that needs discussion. I decided to explicitly import the single class from `mesa.datacollection` because a wildcard import will confuse `flake8`. Should we define a new class in `mesa.datacollection` in the future, we should be able to quickly detect that we need to explicitly include it in this line.